android/app/src/main/AndroidManifest.xml

```diff
@@ -26,12 +26,12 @@
             android:theme="@style/LaunchTheme"
             android:configChanges="orientation|keyboardHidden|keyboard|screenSize|smallestScreenSize|locale|layoutDirection|fontScale|screenLayout|density|uiMode"
             android:hardwareAccelerated="true"
-            android:resizeableActivity="true">
+            android:resizeableActivity="true"
```

Comment:
I think I may have also inadvertently found the issue that was causing #1575! I'm not entirely sure if this is the case, but there was an extra `>` which might've caused `android:windowSoftInputMode="adjustResize"` to not be respected.

Reply:
Nice find!